docs/Install.md

```diff
@@ -2,9 +2,9 @@
 
 ## Introduction
 
-Enarx currently has support for Intel SGX and AMD SEV-SNP. Please check the [Requirements](#requirements).
+Enarx currently has support for Keeps running in Intel SGX and AMD SEV-SNP TEEs (Trusted Execution Environments). Please check the [Requirements](#requirements).  There is also support for application development using a "nil" backend which provides the same runtime, but no security isolation, and can run on a variety of Linux, Mac and Raspberry Pi systems.
```

Comment:
```suggestion
Enarx currently has support for Keeps running in Intel SGX and AMD SEV-SNP TEEs (Trusted Execution Environments). Please check the [Requirements](#requirements). There is also support for application development using a "nil" backend on a variety of Linux, Mac and Raspberry Pi systems, which provides the same runtime, but no security isolation.
```

Reply:
No.  I think the isolation is more important first.

Reply:
me too